### ISSUE TITLE
Intermittent Twitter share and follow verification failures

### ISSUE THREAD

**Issue by franckc** [OriginProtocol/origin#3426]
Looking at the bridge logs, I'm seeing that some Twitter share and follow verifications time out.

Here is a[ link to the filter](https://console.cloud.google.com/logs/viewer?interval=NO_LIMIT&project=origin-214503&minLogLevel=0&expandAll=false&timestamp=2019-09-18T18:30:45.390000000Z&customFacets=&limitCustomFacetWidth=true&advancedFilter=resource.type%3D%22container%22%0Aresource.labels.cluster_name%3D%22origin%22%0Aresource.labels.namespace_id%3D%22prod%22%0Aresource.labels.project_id%3D%22origin-214503%22%0Aresource.labels.zone:%22us-west1-a%22%0Aresource.labels.container_name%3D%22origin-bridge%22%0Aresource.labels.pod_id:%22prod-bridge-%22%0AtextPayload:%22TWITTER%20timed%20out%22&scrollTimestamp=2019-09-18T17:33:12.853383303Z) I'm using.

More specifically, user with account 0x4891c248BD2270D96F9F00B844c57e8C721A1bF5 reported verification did not work for him (he got 0 OGN for sharing).

Some Twitter share verifications do work though (I checked in the bridge logs and in the DB).

**Comment by franckc** [comment]
@shahthepro Any idea what's going on ?

**Comment by franckc** [comment]
These logs seem to indicate the bridge is expecting  a different content than the one it gets from the twitter feed ?

<img width="1577" alt="Screen Shot 2019-09-18 at 11 48 26 AM" src="https://user-images.githubusercontent.com/945910/65176752-4560a300-da0a-11e9-8cfa-aff9b2afae6a.png">

**Comment by shahthepro** [comment]
Seems like the DApp is sending the incorrect tweet for verification. Let me check that.

**Comment by shahthepro** [comment]
Actually the DApp is sending the correct content. But bridge isn't accepting any requests at the moment (returns 504). I guess bridge is overloaded with HTTP requests since we do a (HTTP connection blocking) poll for verification.

**Comment by shahthepro** [comment]
The wrong content might be because the event in the redis being an old one. If bridge is dropping requests, it should be dropping requests from Twitter/Telegram webhooks too.

**Comment by franckc** [comment]
Couple days ago we set VERIFICATION_MAX_TRIES to 30 in EnvKey and bounced the bridge server in prod. It does not look like that helped :(

The [logs](https://console.cloud.google.com/logs/viewer?interval=NO_LIMIT&project=origin-214503&minLogLevel=0&expandAll=false&timestamp=2019-09-21T04:43:35.908000000Z&customFacets=&limitCustomFacetWidth=true&advancedFilter=resource.type%3D%22container%22%0Aresource.labels.cluster_name%3D%22origin%22%0Aresource.labels.namespace_id%3D%22prod%22%0Aresource.labels.project_id%3D%22origin-214503%22%0Aresource.labels.zone:%22us-west1-a%22%0Aresource.labels.container_name%3D%22origin-bridge%22%0Aresource.labels.pod_id:%22prod-bridge-%22%0A&scrollTimestamp=2019-09-20T04:16:11.622507594Z&pinnedLogId=1bqkpc6g36xywsv&pinnedLogTimestamp=2019-09-20T04:16:41.702274681Z) still show some verification failures on Twitter share and follow...

**Comment by shahthepro** [comment]
@franckc I'm looking at the logs again. Seems like we don't receive some of the events. It still works [sometimes](https://console.cloud.google.com/logs/viewer?interval=NO_LIMIT&project=origin-214503&minLogLevel=0&expandAll=false&timestamp=2019-09-21T04%3A43%3A35.908000000Z&customFacets&limitCustomFacetWidth=true&scrollTimestamp=2019-09-20T04%3A18%3A25.173677488Z&advancedFilter=resource.type%3D%22container%22%0Aresource.labels.pod_id%3D%22prod-bridge-6d6864d4b5-dj8hn%22%0Aresource.labels.zone%3D%22us-west1-a%22%0Aresource.labels.project_id%3D%22origin-214503%22%0Aresource.labels.cluster_name%3D%22origin%22%0Aresource.labels.container_name%3D%22origin-bridge%22%0Aresource.labels.namespace_id%3D%22prod%22%0Aresource.labels.instance_id%3D%222797144526571140938%22%0Atimestamp%3D%222019-09-20T04%3A18%3A24.784036750Z%22%0AinsertId%3D%22tf00lvg3afqz37%22). I still feel that bridge is dropping connections. 

Do you we have HTTP incoming logs from nginx? I'd like to take a look at that

**Comment by franckc** [comment]
I think all the nginx ingress logs are [here](https://console.cloud.google.com/logs/viewer?interval=NO_LIMIT&project=origin-214503&minLogLevel=0&expandAll=false&timestamp=2019-09-21T06:02:27.599000000Z&customFacets=&limitCustomFacetWidth=true&advancedFilter=resource.type%3D%22container%22%0Aresource.labels.cluster_name%3D%22origin%22%0Aresource.labels.namespace_id%3D%22prod%22%0Aresource.labels.project_id%3D%22origin-214503%22%0Aresource.labels.zone:%22us-west1-a%22%0Aresource.labels.container_name%3D%22nginx-ingress-controller%22%0Aresource.labels.pod_id:%22prod-ingress-nginx-ingress-controller-%22%0A&scrollTimestamp=2019-09-21T05:23:00.730067633Z).

You can apply a filter like [this](resource.type="container"
resource.labels.cluster_name="origin"
resource.labels.namespace_id="prod"
resource.labels.project_id="origin-214503"
resource.labels.zone:"us-west1-a"
resource.labels.container_name="nginx-ingress-controller"
resource.labels.pod_id:"prod-ingress-nginx-ingress-controller-"
textPayload:"bridge") to only see request going to the bridge.

**Comment by franckc** [comment]
User 0x4891c248BD2270D96F9F00B844c57e8C721A1bF5 with twitter screename gotium is still reporting he is having an issue. He can't seem to verify tweet "New Ways to Earn OGN in September". He says:
```
When i shared this tweet i see "you have rewarded 100 OGN" but i can't see this 100 ogn in my balance. 
```

**Comment by shahthepro** [comment]
@franckc Checking that now. But that was the tweet with a trailing whitespace which I fixed in #3313.

**Comment by shahthepro** [comment]
This should not happen since #3667. Closing this for now.
